crates/web-plugins/oob-messages/src/models.rs

```diff
@@ -112,7 +118,7 @@ where
 
     let did = diddoc.id.clone();
     let oob_message = OobMessage::new(&did);
-    let url: &String = &format!("{}:{}", server_public_domain, server_local_port);
+    let url: &String = &format!("http://{}", server_public_domain);
```

Comment:
appending the http scheme is not necessary here. The `server_public_domain` environment variable will likely embed the right scheme.

Reply:
yes just realized too